worlds/stardew_valley/logic.py

```diff
@@ -1081,8 +1081,7 @@ def has_relationship(self, npc: str, hearts: int = 1) -> StardewRule:
             return self.can_earn_relationship(npc, hearts)
         if self.options[options.Friendsanity] == options.Friendsanity.option_starting_npcs and not villager.available:
             return self.can_earn_relationship(npc, hearts)
-        if self.options[
-            options.Friendsanity] != options.Friendsanity.option_all_with_marriage and villager.bachelor and hearts > 8:
+        if self.options[options.Friendsanity] != options.Friendsanity.option_all_with_marriage and villager.bachelor and hearts > 8:
```

Comment:
for the future: it is preferred to stay within 120 chars per line. pycharm normally does good automated breaking, but sometimes doing a manual break with `\` looks better, like
```python
        if self.options[options.Friendsanity] != \
                options.Friendsanity.option_all_with_marriage and villager.bachelor and hearts > 8:
            ...
```
for example

Reply:
Personally, my Pycharm is configured for 160, not 120, as that is my preference, but I fixed it anyway, as I do agree this one line was difficult to read, and the length was a factor in that